locale/en/docs/index.md

```diff
@@ -42,8 +42,9 @@ This documentation describes the built-in modules provided by Node.js. It does n
 
 ## ES6 features
 
-The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to find which version of V8 shipped with a particular Node.js release.
+The [ES6 section](/en/docs/es6/) describes the three ES6 feature groups, and details which features are enabled by default in Node.js, alongside explanatory links. It also shows how to determine the version of  V8 which shipped with a particular Node.js release.
 
 ## Guides
 
 The [Guides section](/en/docs/guides/) has long-form, in-depth articles about Node.js technical features and capabilities.
+eof: index.md
```

Comment:
This doesn't look right...

Reply:
You're technically correct, traditionally in 'Niad' for ColecoVision's Adam newsletter, we would print a right-justified stinger ****❇️ at the bottom but I believe the modern recognized signoff is end-of-file {filename}
**🏵**